frontend/freebase2wikidata.js

```diff
@@ -34,9 +34,9 @@
 */
 
 $(function() {
-  'use strict';
 
   var async = module.exports;
+  var ps = mw.ps || {};
```

Comment:
I initially put the functions in the closure, as you suggested. The problem is that they are called from an onClick of a button, and they are not visible from outside of the closure. If you have any suggestion on how to fix this I could remove the global object.

Reply:
I see. Instead of building the HTML from a string and `on*` attributes you could maybe build it using jQuery or something similar, allowing you to call functions declared in the closure. Eg.

```
$('<div>').attr('id', 'myNav').addClass('overlay')
   .append(
             $('<a>').addClass('closebtn).click(closeNav),
             $('<div>').attr('id', "blackboard").addClass("overlay-content")
  );
```

It's probably even going to make more sense if you use OOjs UI.